### PR TITLE
update cypress translations

### PR DESCRIPTION
**Pull request by Yibaebi**

### CODE REVIEW

File: cypress.config.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ export default defineConfig({
       hideCredentials: true,
       TEST_USERNAME: '***',
       TEST_PASSWORD: '***',
+      TEST_GOOGLE_PASSWORD: '***',
```
Comment (future-pirate-king) [inline cypress.config.ts:14]: why do we require this ?

File: cypress/support/translations.ts [hunk 2]
```diff
@@ -1,7 +1,56 @@
...
+}
+
+/**
+ * Translates a message using the provided key, substitutes variables, and formats it based on locale.
+ * @param {NestedKeyOf<typeof EN_TRANSLATIONS>} key - The key to retrieve the message value.
+ * @param {Record<string, string | number>} [variables] - Optional variables to substitute in the message.
+ * @param {keyof typeof MESSAGES_MAP} [locale] - Optional locale to format the message. Defaults to 'en' if not provided.
+ * @returns {string} The translated and formatted message.
+ */
+export function cypressTranslate(
```
Comment (future-pirate-king) [inline cypress/support/translations.ts:45]: can we rename this to `cyTranslate` and can we remove `export` as it is the default export

File: cypress/tsconfig.json [hunk 3]
```diff
@@ -7,6 +7,7 @@
     "lib": ["es2015", "dom", "es2018.regexp"],
     "isolatedModules": false,
     "allowJs": true,
-    "noEmit": true
+    "noEmit": true,
+    "moduleResolution": "node"
```
Comment (future-pirate-king) [inline cypress/tsconfig.json:11]: why is this required ?

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: cypress.config.ts (hunk 1 above)
Comment (Yibaebi) [reply cypress.config.ts:14]: It's not required for this PR. I'll take it off.

**Review by future-pirate-king — APPROVED** [review]
